stories/topics/tech-note-gcp-secrets-update.adoc

```diff
@@ -0,0 +1,5 @@
+[id="tech-note-gcp-secrets-update"]
+
+= Secrets Update
+
+When you update any secrets in GCP secret manager, ensure latest secret version is enabled. For example, if you have 2 secret versions for a `<deployment-name>-aap-admin` secret, secret version 2 must be enabled where `<deployment-name>` is the name of your foundation deployment.
```

Comment:
in the GCP secret manager, ensure that the latest secret...

For example, if you have two secret versions... must be enabled, where ....